tools/check_format.py

```diff
@@ -196,8 +197,35 @@ def hasInvalidAngleBracketDirectory(line):
   return subdir in SUBDIR_SET
 
 
+VERSION_HISTORY_NEW_LINE_REGEX = re.compile('\* [a-z \-_]*: [a-z:`]')
+
+
+def checkVersionFileContents(file_path, error_messages):
+  file_handle = fileinput.input(file_path)
+  for line_number, line in enumerate(file_handle):
+
+    def reportError(message):
+      error_messages.append("%s:%d: %s" % (file_path, line_number + 1, message))
+
+    # We stop line checks at a somewhat arbitrary point to avoid rewriting very old release notes
+    # to the new format.  If we get to the point a single Envoy release is more than 400 lines we'll
+    # have to adjust this for it to be effective.
+    if line_number > 400:
```

Comment:
It should be pretty easy to find then end of the current release stanza by looking for "===============" or similar in this loop. That seems more future-proof than checking for 400.